distributed/client.py

```diff
@@ -1090,7 +1090,8 @@ def _close(self, fast=False):
             # This makes the shutdown slightly smoother and quieter
             with ignoring(AttributeError, gen.TimeoutError):
                 yield gen.with_timeout(timedelta(milliseconds=100),
-                                       self._handle_scheduler_coroutine)
+                                       self._handle_scheduler_coroutine,
+                                       CancelledError)
```

Comment:
Nitpicks:
* maybe add the parameter name for the last argument since it is optional in `gen.with_timeout`
* maybe use a tuple of length 1 to be closer to the `gen.with_timeout` signature (using `CancelledError` works because the argument is only used in a `isinstance` call)

i.e. something like this:
```py
yield gen.with_timeout(..., quiet_exceptions=(CancelledError,))`
```